routing/portal_config.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/*
+ * As a work of the United States government, this project is in the public domain within the United States.
+ */
+
```

Comment:
Should this also include namespace for consistency?

Reply:
This is certainly something we can add, but for this all I did was mirror the way it was set up in the original configs. I was trying to keep the changes simple, so that the changes to the wider codebase could be kept to a minimum.